api/src/main/java/io/opencensus/trace/Tracer.java

```diff
@@ -169,51 +175,50 @@ public final SpanBuilder spanBuilder(String name) {
    *
    * <p>See {@link SpanBuilder} for usage examples.
    *
-   * <p>This <b>must</b> be used to create a {@code Span} when manual Context propagation is used
-   * OR when creating a root {@code Span} with a {@code null} parent.
+   * <p>This <b>must</b> be used to create a {@code Span} when manual Context propagation is used OR
+   * when creating a root {@code Span} with a {@code null} parent.
    *
-   * @param parent The parent of the returned Span. If null the {@code SpanBuilder} will build a
-   *     root {@code Span}.
-   * @param name The name of the returned Span.
+   * @param spanName The spanName of the returned Span.
+   * @param parent The parent of the returned Span. If {@code null} the {@code SpanBuilder} will
+   *     build a root {@code Span}.
    * @return a {@code SpanBuilder} to create and start a new {@code Span}.
-   * @throws NullPointerException if {@code name} is null.
+   * @throws NullPointerException if {@code spanName} is {@code null}.
    */
-  public abstract SpanBuilder spanBuilder(@Nullable Span parent, String name);
+  public abstract SpanBuilder spanBuilderWithParent(String spanName, @Nullable Span parent);
```

Comment:
Shouldn't this be something like `spanBuilderWithExplicitParent`, since `spanBuilder` also creates the SpanBuilder with a parent, if there is one in the Context?